tests/helpers.py

```diff
@@ -61,6 +63,25 @@ def open(self, *args, **kwargs):
         return super(CTFdTestClient, self).open(*args, **kwargs)
 
 
+class FreezeTime(str, RawEnum):
+    NOT_STARTED = "2017-10-3"  # Tuesday, October 3, 2017
+    STARTED = "2017-10-5"  # Thursday, October 5, 2017
+    ENDED = "2017-10-7"  # Saturday, October 7, 2017
```

Comment:
This should be moved into a constants file/directory in the tests so we don't overcrowd the helpers file.